### PR TITLE
bootstrap a community page

### LINKED ISSUE #1086: Add a dedicated page for community rules
To help community to explore them
refs: https://github.com/ohze/scalafix-rules
and:
https://github.com/NeQuissimus/sort-imports

### PR DESCRIPTION
**Pull request by mlachkar**
fixes #1086
image

### CODE REVIEW

**Comment by vincenzobaz** [comment]
I like the page!
The [landing page](https://scalacenter.github.io/scalafix/) contains 3 "elements" (Refactoring, Linting, Enforce in CI).
I think it would be nice to add a 4th one called something like "Extensible" which has a link to the page "Community rules"

**Comment by fthomas** [comment]
I like this page too and was wondering if it also makes sense to link to Scala Steward's list of rules:
* https://github.com/scala-steward-org/scala-steward/blob/master/docs/scalafix-migrations.md
* https://github.com/scala-steward-org/scala-steward/blob/master/modules/core/src/main/resources/scalafix-migrations.conf

File: docs/rules/Community-rules.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+title: Community rules
+---
+
+Many rules have been developed by the community.
+Follow [this documentation](external-rules.md) to use them in your project.
+
+
+
+## Available rules:
+*[Edit](https://github.com/scalacenter/scalafix/edit/master/docs/developers/cross-publish-custom-rules.md)
```
Comment (bjaglin) [inline docs/rules/Community-rules.md:13]:
```suggestion
*[Edit](https://github.com/scalacenter/scalafix/edit/master/docs/rules/Community-rules.md) 
```

Comment (bjaglin) [reply docs/rules/Community-rules.md:13]: maybe the file name could be lower case to be consistent with other non-rule doc pages?

Comment (mlachkar) [reply docs/rules/Community-rules.md:13]: ok.

**Comment by mlachkar** [comment]
@vincenzobaz it's interesting to modify the landing page, but I will do it in another PR.
 @fthomas I really like this feature in Scala Steward. I wasn't aware of it. I don't know yet how to link the scala Steward rules. There are not published and can only be used with scalafix command line. 
 I will do that in another PR.
 @bjaglin is all of this ok for you ? 
 Thanks for the reviews and propositions.

File: docs/rules/Community-rules.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+sidebar_label: Community rules
+title: Community rules
+---
+
+Many rules have been developed by the community.
+Follow [this documentation](external-rules.md) to use them in your project.
+
+
+## Available rules:
+*[Edit](https://github.com/scalacenter/scalafix/edit/master/docs/rules/community-rules.md)
```
Comment (bjaglin) [inline docs/rules/Community-rules.md:12]: looks like the file name is still upper-case, does it work?

Comment (mlachkar) [reply docs/rules/Community-rules.md:12]: ah thanks

**Review by bjaglin — APPROVED** [review]
:+1:

**Pull request merged**